modules/ROOT/pages/bat-bdd-reference.adoc

```diff
@@ -84,6 +84,70 @@ describe `Demo scenario` in [
 
 A test suite must contain at least one test (.dwl) file.
 
+== Keeping Confidential Information Secure in Tests
+
+In your tests, you can access shared secrets in Anypoint Secrets Manager, which allows you to store securely sensitive information, such as a password, authentication token, endpoint URL, webhook URL, or something else. You create aliases for your shared secrets with the `bat grant` command. The BAT CLI stores aliases in your `bat.yaml` files in your test suites. You place aliases in your tests and, at runtime, the BAT CLI looks up the shared secrets by means of the aliases.
+
+[IMPORTANT]
+====
+In a given test suite, you can use aliases for shared secrets only if you plan to run the tests in that suite from a private location.
+====
+
+. Store the information () as a shared secret in Anypoint Secrets Manager.
+. In Anypoint Secrets Manager, create a shared secret as a symmetric key. If the info that you want to store as a shared secret is currently in plain text, then you must encode it in Base64 when you create the secret.
+a. Open a secrets group or create a new one. The group must be created in the same environment that you are using in the BAT CLI. To find out which environment you are currently using in the BAT CLI, run the command `bat whoami`. In the output, there is the ID for the environment. Run the command `bat environment ls` to list the environments that you have access to. Match the ID from the `bat whoami` command with one of the environments listed. If you need to switch to the environment that your secrets group is in, run the command `bat environment switch name`, where `name` is the name of the environment.
+b. Select Shared Secret on the left side of the screen.
+c. In the *Type* field, select *Symmetric Key*.
+d. In the *Key* field, paste your license key encoded as a Base64 string.
+e. Paste the same Base64 string into the *Confirm Key* field.
+. Copy the name of the new shared secret.
+. At a command prompt, run the `bat grant` command, specifying an alias for the shared secret. When you run this command, the BAT CLI creates a section named `secrets` in your test suite's `bat.yaml` file, if the section does not already exist. In that section, the BAT CLI adds these indented lines:
```

Comment:
```suggestion
. At a command prompt, run the `bat grant` command, specifying an alias for the shared secret. 
When you run this command, the BAT CLI creates a section named `secrets` in your test suite's `bat.yaml` file, if the section does not already exist. In that section, the BAT CLI adds these indented lines:
```

Reply:
Do we really have to document what the CLI does in the background?
AND the syntax values involved?
As with Step 1a., I think this information probably belongs somewhere outside of the task steps....

Reply:
Yes, we do need to document that. These topics are for the manual process of creating test suites. Users manually create the bat.yaml file for their test suite, so they need to know why content is added to it, so that they are not surprised by it.